
<launch>
    <include file="$(find roborts_bringup)/launch/base.launch" />
   
    <node pkg="imu6050" type="imu6050_node" name="imu6050_node" output="screen">
        <param name="serial_port" type="string" value="/dev/imu_serial"/>
        <param name="serial_baudrate" type="int" value="1000000"/>
        <param name="frame_id" type="string" value="imu_link"/>
        <param name="topic_name" type="string" value="imu"/>
    </node>
    <!-- <node pkg="tf" type="static_transform_publisher" name="tf_broadcaster" args="0.065 0.0 0.0 0.0 0.0 0.0 0.0 /base_link /imu_link 100" /> -->
</launch>
